deploy/kuberhealthy-prometheus-operator.yaml

```diff
@@ -301,9 +328,6 @@ metadata:
   name: kuberhealthy
   namespace: kuberhealthy
   annotations:
-    prometheus.io/scrape: "true"
-    prometheus.io/port: "80"
-    prometheus.io/path: "/metrics"
```

Comment:
Why drop the prometheus scrape annotations here?  I would prefer to scrape the service for the global state of checks rather than checking each pod individually; having them overwrite each others information.

Reply:
There's a `ServiceMonitor` in `kuberhealthy-prometheus-operator.yaml` already. These annotations had no effect here. We only need these in `kuberhealthy-prometheus.yaml`.